### ISSUE TITLE
It's Genitive not Genetive

### ISSUE THREAD

**Issue by alex-mashin** [wapmorgan/Morphos#1]
Subj.

**Comment by githubjeka** [comment]
линейкы
```php
var_dump($dec->getForms('линейка', false));
/* Will produce something like
  array(6) {
    ["nominativus"]=>
    string(14) "линейка"
    ["genetivus"]=>
    string(14) "линейкы"
    ["dativus"]=>
    string(14) "линейке"
    ["accusative"]=>
    string(14) "линейку"
    ["ablativus"]=>
    string(16) "линейкой"
    ["praepositionalis"]=>
    string(17) "о линейке"
  }
*/
```

**Comment by wapmorgan** [comment]
@githubjeka, Strange. There's support for this. Did you installed latest version of Morphos?

**Comment by wapmorgan** [comment]
@alex-mashin, OK. Thanks. We'll be fixed soon.

**Comment by githubjeka** [comment]
@wapmorgan It's from README.md

**Comment by wapmorgan** [comment]
@alex-mashin, fixed in 2.2.9.
